docs/create_robots_whoami.md

```diff
@@ -30,11 +30,12 @@ Your robot's `whoami` package serves as a configuration package for the `rai_who
 4. Run the `parse_whoami_package`. This will process the documentation, building it into a vector database, which is used by RAI agent to reason about its identity.
 
 > [!IMPORTANT]
-> Parsing bigger documents might lead to costs. Embedding model can be configured in
+> Parsing bigger documents with Cloud vendors might lead to costs. Consider using the
+> local `ollama` provider for this task. Embedding model can be configured in
```

Comment:
So if the embedding model can be configured here, why would someone specify the vendor later?

Reply:
https://github.com/RobotecAI/rai/pull/240/commits/3bf785669d351a1c5a47f2d6b4688f4e9adb6232